sensor/common/scan/scan.go

```diff
@@ -2,10 +2,11 @@ package scan
 
 import (
 	"context"
+	"errors"
 	"fmt"
 	"time"
 
-	"github.com/pkg/errors"
+	ghErrors "github.com/pkg/errors"
```

Comment:
We've called it `pkgErrors` in other cases where there was a conflict, maybe that's more clear to people but I don't really have a strong opinion on it 🤷

Reply:
Thats an excellent point, will make the update!